blocks/marketo/marketo.css

```diff
@@ -0,0 +1,336 @@
+/* stylelint-disable selector-id-pattern */
+/* stylelint-disable selector-class-pattern */
+.marketo {
+    width: 100%;
+}
+
+/* Merative styling for Marketo form */
+.mktoForm {
```

Comment:
😮 Why is everything `!important`? Is this a Marketo thing?

Reply:
yeah i got the css from their current form. Basically what is happening is that there is a form that gets embedded when we load this script. and that pulls in JS that delivers the form in a certain style and we are styling that embedded form to overwrite their existing styles. I can do some testing to see if i can get away with it though. let me check

Reply:
yeah, so the marketo form comes with it's own CSS that needs to be overridden this way unfortunately.